app/helpers/application_helper.rb

```diff
@@ -135,8 +130,9 @@ def follow_button(followable, style = "full", classes = "")
   end
 
   def user_colors_style(user)
-    "border: 2px solid #{user.decorate.darker_color}; \
-    box-shadow: 5px 6px 0px #{user.decorate.darker_color}"
+    darker_color = user.decorate.darker_color
```

Comment:
Okay, this one came along for the ride, we were calling `user.decorate` twice.  Which was unnecessary